packages/block-editor/src/components/inserter/preview-panel.js

```diff
@@ -38,9 +47,18 @@ function InserterPreviewPanel( { item } ) {
 					<div className="block-editor-inserter__preview-content">
 						<BlockPreview
 							blocks={ blocks }
-							viewportWidth={ example?.viewportWidth ?? 500 }
+							viewportWidth={ viewportWidth }
+							minHeight={ previewHeight }
 							additionalStyles={ [
-								{ css: 'body { padding: 24px; }' },
+								{
+									css: `
+										body { 
+											padding: 24px;
+											min-height:${ Math.round( minHeight ) }px;
+											display:flex;align-items:center;justify-content:center; 
+										}
```

Comment:
```suggestion
										}
										.is-layout-grid { width: 100vw; }
```

@jasmussen from my tests we would need something like this (100% wouldn't work because there's another wrapper div that is already the wrong size)